usagov/vote-gov-drupal
# VOTE-292 Bug - Navajo time display

## Jira ticket

[VOTE-292
](https://cm-jira.usa.gov/browse/VOTE-292)

## Description
To correct the display of the time, override the Navajo font for the registration double field time

## Deployment and testing

### Pre-deploy

1. lando retune
2. You might have to also build the vote_gov styles

### Post-deploy

1. Log into the CMS
2. Edit a state page to use an exact date and time for online and in person registration.

### QA/Test

1. View the state page edited in the post deploy step (remain logged in)
2. Switch to Navajo, and confirm the time displays correctly
3. Confirm the entire date is displayed in the same line as the label

## Checklist for the Developer




## Checklist for the Peer Reviewers
